selfdrive/car/hyundai/carstate.py

```diff
@@ -141,7 +142,7 @@ def update(self, cp, cp_cam):
 
     ret.gearShifter = self.parse_gear_shifter(self.shifter_values.get(gear))
 
-    if not self.CP.openpilotLongitudinalControl:
+    if not self.CP.openpilotLongitudinalControl and not (self.CP.flags & HyundaiFlags.CAN_CANFD.value):
```

Comment:
```suggestion
    # TODO: check AEB and FCW on CAN-CAN FD cars
    if not self.CP.openpilotLongitudinalControl and not (self.CP.flags & HyundaiFlags.CAN_CANFD):
```

is `CAN-CAN FD` a good name in the comments for these cars?